salt/pillar/extra_minion_data_in_pillar.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+'''
+Add all extra minion data to the pillar.
+
+:codeauthor: Alexandru.Bleotu@morganstanley.ms.com
+
+One can filter on the keys to include in the pillar by using the ``include``
+parameter. For subkeys the ':' notation is supported (i.e. 'key:subkey')
+The keyword ``<all>`` includes all keys.
+
+Complete example in etc/salt/master
+=====================================
+
+.. code-block:: yaml
+
+    ext_pillar:
+      - extra_minion_data_in_pillar:
+          include: <all>
```

Comment:
IMHO, this should be `include: '*'` to make it more like the typical salt targeting syntax.

Reply:
Makes sense. Ok if I do both? It's easier for us as we have been using `<all>`

Reply:
Of course. Both is just fine. Thanks!